docker/ci/install/install-rocksdb

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# RocksDB dependencies
+# Install gflags
```

Comment:
Are you sure we need gflags and zstandard? Especially if we're going to just install snappy and rocksdb from pre-built binaries.

Also, I'm pretty sure I've been building this locally without them.